src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitArray = sourceString.trim().split(';');
+  const object = {};
```

Comment:
The variable name 'object' is too generic. Use a more descriptive name, such as 'styleObject', to improve code readability.